src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/TableInsertAttempt.java

```diff
@@ -43,11 +44,10 @@ public Optional<ColumnsDescContainer> schemaDescription() {
     if (row == null) {
       return Optional.empty();
     }
-
     var apiColumns = schemaObject.apiTableDef().primaryKeys();
-    if (!apiColumns.filterByUnsupported().isEmpty()) {
-      throw new IllegalStateException(
-          "Unsupported columns primary key: %s" + apiColumns.filterByUnsupported());
+    var unsupported = apiColumns.filterBySupport(ApiSupportDef::isUnsupportedInsert);
```

Comment:
Fix required to find columns that do not support Insert and NOT columns that miss support for any of operations -- f.ex. `time_uuid` does support Insert but not Create (table).
Without fix added test fails so the fix is added via this PR.